nexus/src/app/background/blueprint_execution.rs

```diff
@@ -71,6 +78,10 @@ impl BackgroundTask for BlueprintExecutor {
             )
             .await;
 
+            // Trigger anybody waiting for this to finish.
+            self.count = self.count + 1;
+            self.tx.send(self.count);
```

Comment:
I'll take "`watch` is hard to use correctly for $200, Alex"; this should probably be `send_replace` or `send_modify` so it doesn't spuriously fail if there are no subscribers.

Reply:
Actually, if we used `send_modify` could we drop `self.count` altogether?

```rust
self.tx.send_modify(|count| *count += 1)

Reply:
Good call!

Reply:
Fixed in #70dfed3d